extensions/amp-analytics/0.1/analytics-root.js

```diff
@@ -142,6 +142,21 @@ export class AnalyticsRoot {
   getElementById(unusedId) {}
 
   /**
+   * Returns the tracker for the specified name and list of allowed types.
+   *
+   * @param {string} name
+   * @param {!Object<string, function(new:./events.EventTracker)>} options
+   * @return {?./events.EventTracker}
+   */
+  getTrackerForOptions(name, options) {
```

Comment:
The option here is serving as a klass whitlist. would `whitelistKlass` be a better name?